6MI0800239/pract.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+void readFile(ifstream& ifs,int& sum,int& proiz)
+{
+    ifstream file("result.txt");
+
+    if(!file.is_open())
+    {
+        return;
+    }
+
+    file >> sum >> proiz;
+}
+
+//int main()
+//{
+//    int a;
+//    int b;
+//    int c;
+//
+//    cin >> a >> b >> c;

```

Comment:
It'll be nice to include some user prompts here to help understand what's expected from the user :P